src/cmd-buildextend-installer

```diff
@@ -288,7 +288,8 @@ def generate_iso():
             run_verbose(['rm', '-rf', os.path.join(tmpisoroot, d)])
 
         # grub2-mkrescue is a wrapper around xorriso
-        genisoargs = ['grub2-mkrescue']
+        genisoargs = ['grub2-mkrescue',
+                     '-volid', volid]
```

Comment:
A nit so minor might be classified by physicists as having no mass:

Why the line break?

Reply:
looking at other places where genisoargs are defined with arguments being in a separate line, i wanted to keep things in line with that, but sure..i can do away with the line break

Reply:
Not a big deal, I'll give you a break on this one

Reply:
done. had to make changes anyway.